### DISCUSSION TITLE
Code tutorial: Transcribe pre-recorded audio missing an option

### DISCUSSION THREAD

**Discussion by GregStevenson** [deepgram/community#309] [General help]
In the .Net code,  PreRecordedTranscriptionOptions is missing Utterances = true and raises and exception. May be true for other language examples as well. Exception message was helpful enough for me to figure it out and get project working, but should be fixed in tutorial code.

**Comment by jkroll-deepgram** [comment]
Hi @GregStevenson, are you referring to the .NET SDK, or a specific tutorial? If the SDK, what version are you using? The SDK is intended to support utterances: https://github.com/deepgram/deepgram-dotnet-sdk/blob/main/Deepgram/Models/PrerecordedTranscriptionOptions.cs#L188

Since it sounds like you made code changes to get it working, please feel free to open a PR or share the changes you made in order to resolve any problems around utterances. It would be informative to see where/how the exception was being raised too. Thanks for your help in improving the code and developer experience.
